src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
+$main-accent-color: #060b35;
+$secondary-accent-color: #616070;
+$blue-accent-color: #00acdc;
+$white-accent-color: #fff;
+$font-family: Roboto, sans-serif;
+
+@import '../utils/veriables';
+@import '../styles/blocks/card';
+@import '../styles/blocks/stars';
```

Comment:
Verify that the paths for the imported SCSS files are correct. Ensure that `../styles/blocks/card` and `../styles/blocks/stars` exist and are correctly named.